components/dashboard/src/settings/Teams.tsx

```diff
@@ -484,7 +484,7 @@ function AllTeams() {
                                 Payment in Progress
                             </div>
                         </div>
-                        <div className="my-auto flex w-1/12 pl-8 opacity-0 group-hover:opacity-100">
+                        <div className="my-auto flex w-1/12 opacity-0 group-hover:opacity-100 justify-end">
```

Comment:
@gtsiolis, I know there is no tester for this change in core :-(
but you can compare the styles with the environment variables page. it's just aligned.